Mattermost/Info.plist

```diff
@@ -46,6 +46,10 @@
 		<key>NSAllowsArbitraryLoads</key>
 		<true/>
 	</dict>
+	<key>NSCameraUsageDescription</key>
+	<string>Attach photo and video from camera</string>
+	<key>NSPhotoLibraryUsageDescription</key>
+	<string>You can safely ignore and discard this</string>
```

Comment:
this was actually very fun, but it turns out to be a bug at Apple side: there is a standard UI to pick up photos from photo library without requesting any permission, but you requested it from the webview and it turns out to work differently. really works for me even if I deny permission for the photo library usage.